src/controllers/quotes.js

```diff
@@ -23,6 +23,47 @@ class QuotesController {
       next(e);
     }
   }
+
+  /**
+   * Adds a new quote to the system
+   * @param req Request object
+   * @param res Reponse object
+   * @param next The next middleware
+   * @returns {Promise<>}
+   */
+  static async add(req, res, next) {
+    try {
+      // Get the parameters from the body of the HTTP request
+      const {
+        playName,
+        quoteText,
+        lineNumber,
+        speaker,
+      } = req.body;
+
```

Comment:
You also have a syntax error with your const. It should be written const req.body = {}